### PR TITLE
Add Storybook Style Recommendations

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

**Review by caroqliu — COMMENTED** [review]
This looks great! Thank you for writing it up!

One note we could add here is to title the component names in `--storybook_env=preact` without Bento prefix for easier navigability between components.

**Review by caroqliu — APPROVED** [review]

File: build-system/tasks/storybook/README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+# Storybook
+
+## Style Recommendations
+
+### Do not use Knobs
```
Comment (samouri) [inline build-system/tasks/storybook/README.md:5]: This first bit about knobs feels more like it should be an announcement to active contributors than a permanent part of the repo.

**Review by samouri — COMMENTED** [review]
These style recommendations LGreatToMe

File: build-system/tasks/storybook/README.md (hunk 1 above)
Comment (alanorozco) [reply build-system/tasks/storybook/README.md:5]: Fair enough! I'll be notifying maintainers and adding a lint rule (#35991)

**Pull request merged**